packages/element/src/test/create-interpolate-element.js

```diff
@@ -207,19 +207,19 @@ describe( 'createInterpolateElement', () => {
 				</div>
 			);
 		};
-		let renderer;
-		act( () => {
-			renderer = TestRenderer.create(
-				<TestComponent switchKey={ true } />
-			);
-		} );
-		expect( () => renderer.root.findByType( 'em' ) ).not.toThrow();
-		expect( () => renderer.root.findByType( 'strong' ) ).toThrow();
-		act( () => {
-			renderer.update( <TestComponent switchKey={ false } /> );
-		} );
-		expect( () => renderer.root.findByType( 'strong' ) ).not.toThrow();
-		expect( () => renderer.root.findByType( 'em' ) ).toThrow();
+		const { container, rerender } = render(
+			<TestComponent switchKey={ true } />
```

Comment:
We could use the opportunity to remove the redundant `={ true }`.

```suggestion
			<TestComponent switchKey />
```